Some unit tests failed in primitives/src/chain/spec.rs.

### Describe the bug

When trying to run a unit test function in file `primitives/src/chain/spec.rs`., for example [test_fork_timestamps()](https://github.com/paradigmxyz/reth/blob/main/crates/primitives/src/chain/spec.rs#L1695) and [ignores_genesis_fork_blocks()](https://github.com/paradigmxyz/reth/blob/main/crates/primitives/src/chain/spec.rs#L1735), the following problem was encountered:

```
Compiling reth-primitives v0.1.0-alpha.13 
error[E0277]: the trait bound `HashBuilderValue: _IMPL_ARBITRARY_FOR_Withdrawal::_proptest::arbitrary::Arbitrary` is not satisfied
```


### Steps to reproduce

1. ` cargo test --package reth-primitives --lib -- chain::spec::tests::ignores_genesis_fork_blocks --exact --nocapture`
2.  `cargo test --package reth-primitives --lib -- chain::spec::tests::test_fork_timestamps --exact --nocapture `

### Node logs

```text
error[E0277]: the trait bound `HashBuilderValue: _IMPL_ARBITRARY_FOR_Withdrawal::_proptest::arbitrary::Arbitrary` is not satisfied
  --> crates/primitives/src/trie/hash_builder/state.rs:13:12
   |
13 | pub struct HashBuilderState {
   |            ^^^^^^^^^^^^^^^^ the trait `_IMPL_ARBITRARY_FOR_Withdrawal::_proptest::arbitrary::Arbitrary` is not implemented for `HashBuilderValue`
   |
   = help: the following other types implement trait `_IMPL_ARBITRARY_FOR_Withdrawal::_proptest::arbitrary::Arbitrary`:
             ()
             (Sender<A>, std::sync::mpsc::IntoIter<A>)
             (Sender<A>, std::sync::mpsc::Receiver<A>)
             (SyncSender<A>, std::sync::mpsc::IntoIter<A>)
             (SyncSender<A>, std::sync::mpsc::Receiver<A>)
             (T0, T1)
             (T0, T1, T2)
             (T0, T1, T2, T3)
           and 237 others
   = note: this error originates in the derive macro `HashBuilderStatePropTestArbitrary` (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0277]: the trait bound `TrieMask: _IMPL_ARBITRARY_FOR_Withdrawal::_proptest::arbitrary::Arbitrary` is not satisfied
  --> crates/primitives/src/trie/hash_builder/state.rs:13:12
   |
13 | pub struct HashBuilderState {
   |            ^^^^^^^^^^^^^^^^ the trait `_IMPL_ARBITRARY_FOR_Withdrawal::_proptest::arbitrary::Arbitrary` is not implemented for `TrieMask`
   |
   = help: the following other types implement trait `_IMPL_ARBITRARY_FOR_Withdrawal::_proptest::arbitrary::Arbitrary`:
             ()
             (Sender<A>, std::sync::mpsc::IntoIter<A>)
             (Sender<A>, std::sync::mpsc::Receiver<A>)
             (SyncSender<A>, std::sync::mpsc::IntoIter<A>)
             (SyncSender<A>, std::sync::mpsc::Receiver<A>)
             (T0, T1)
             (T0, T1, T2)
             (T0, T1, T2, T3)
           and 237 others
   = note: required for `Vec<TrieMask>` to implement `_IMPL_ARBITRARY_FOR_Withdrawal::_proptest::arbitrary::Arbitrary`
   = note: this error originates in the derive macro `HashBuilderStatePropTestArbitrary` (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0277]: the trait bound `HashBuilderValue: _IMPL_ARBITRARY_FOR_Withdrawal::_proptest::arbitrary::Arbitrary` is not satisfied
  --> crates/primitives/src/trie/hash_builder/state.rs:18:5
   |
18 |     /// The current node value.
   |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^ the trait `_IMPL_ARBITRARY_FOR_Withdrawal::_proptest::arbitrary::Arbitrary` is not implemented for `HashBuilderValue`
   |
   = help: the following other types implement trait `_IMPL_ARBITRARY_FOR_Withdrawal::_proptest::arbitrary::Arbitrary`:
             ()
             (Sender<A>, std::sync::mpsc::IntoIter<A>)
             (Sender<A>, std::sync::mpsc::Receiver<A>)
             (SyncSender<A>, std::sync::mpsc::IntoIter<A>)
             (SyncSender<A>, std::sync::mpsc::Receiver<A>)
             (T0, T1)
             (T0, T1, T2)
             (T0, T1, T2, T3)
           and 237 others

error[E0277]: the trait bound `TrieMask: _IMPL_ARBITRARY_FOR_Withdrawal::_proptest::arbitrary::Arbitrary` is not satisfied
  --> crates/primitives/src/trie/hash_builder/state.rs:25:5
   |
25 |     /// Hash masks.
   |     ^^^^^^^^^^^^^^^ the trait `_IMPL_ARBITRARY_FOR_Withdrawal::_proptest::arbitrary::Arbitrary` is not implemented for `TrieMask`
   |
   = help: the following other types implement trait `_IMPL_ARBITRARY_FOR_Withdrawal::_proptest::arbitrary::Arbitrary`:
             ()
             (Sender<A>, std::sync::mpsc::IntoIter<A>)
             (Sender<A>, std::sync::mpsc::Receiver<A>)
             (SyncSender<A>, std::sync::mpsc::IntoIter<A>)
             (SyncSender<A>, std::sync::mpsc::Receiver<A>)
             (T0, T1)
             (T0, T1, T2)
             (T0, T1, T2, T3)
           and 237 others
   = note: required for `Vec<TrieMask>` to implement `_IMPL_ARBITRARY_FOR_Withdrawal::_proptest::arbitrary::Arbitrary`

For more information about this error, try `rustc --explain E0277`.
error: could not compile `reth-primitives` (lib test) due to 4 previous errors
```


### Platform(s)

Mac (Apple Silicon)

### What version/commit are you on?

861586362053513115e031df3db2661a48e2ff9c

### What database version are you on?

_No response_

### What type of node are you running?

None

### What prune config do you use, if any?

_No response_

### If you've built Reth from source, provide the full command you used

_No response_

### Code of Conduct

- [X] I agree to follow the Code of Conduct

--- a/crates/primitives/Cargo.toml
+++ b/crates/primitives/Cargo.toml
@@ -69,6 +69,8 @@ proptest.workspace = true
 proptest-derive.workspace = true
 rand.workspace = true
 revm-primitives = { workspace = true, features = ["arbitrary"] }
+nybbles = { version = "0.1", features = ["arbitrary"] }
+alloy-trie = { version = "0.1", features = ["arbitrary"] }
 serde_json.workspace = true
 test-fuzz = "4"
 toml.workspace = true

